test/db/formats/elf/reloc

```diff
@@ -90,38 +88,50 @@ pd 1 @ 0x0800008c
 pd 1 @ 0x0800009f
 EOF
 EXPECT=<<EOF
-0x0800007c      e88e0a0000     call  reloc.target.__fentry
-0x0800008c      e8860a0000     call  reloc.target.printk
-0x0800009f      e8730a0000     call  reloc.target.printk
+0x0800007c      e8970a0000     call  reloc.target.__fentry
+0x0800008c      e88f0a0000     call  reloc.target.printk
+0x0800009f      e87c0a0000     call  reloc.target.printk
 --
-0x0800007c      e88e0a0000     call  __fentry__
-0x0800008c      e8860a0000     call  printk
-0x0800009f      e8730a0000     call  printk
+0x0800007c      e8970a0000     call  __fentry__
+0x0800008c      e88f0a0000     call  printk
+0x0800009f      e87c0a0000     call  printk
 EOF
 RUN
 
-NAME=ELF: RZ_X86_64_PLT32 patch reloc 2 (no io.cache)
-FILE=bins/elf/test.ko
+NAME=ELF: no patching when bin.relocs=0
+FILE=bins/elf/analysis/filetime.c-clang-x64-O0.o
+ARGS=-e bin.relocs=false
 CMDS=<<EOF
+om
+?e --
+e asm.flags.real=false
 e asm.bytes=true
-e asm.comments=false
-e asm.lines=false
-e asm.flags=false
-pd 1 @ 0x0800007c
-pd 1 @ 0x0800008c
-pd 1 @ 0x0800009f
+f@F:relocs
+pd 1 @ 0x08000079
+pd 1 @ 0x080000a2
+pd 1 @ reloc.target.strcmp
 EOF
 EXPECT=<<EOF
-0x0800007c      e800000000     call  __fentry__
-0x0800008c      e800000000     call  printk
-0x0800009f      e800000000     call  printk
+ 9 fd: 3 +0x00000e00 0x08000e00 - 0x08000ef2 --- fmap..strtab
+ 8 fd: 3 +0x00000040 0x08000040 - 0x08000635 r-x fmap..text
+ 7 fd: 3 +0x00000b00 0x08000b00 - 0x08000d6f --x fmap..rela.text
+ 6 fd: 3 +0x00000636 0x08000636 - 0x08000824 r-- fmap..rodata.str1.1
+ 5 fd: 3 +0x00000825 0x08000825 - 0x0800085b --- fmap..comment
+ 4 fd: 3 +0x00000860 0x08000860 - 0x0800091f r-- fmap..eh_frame
+ 3 fd: 3 +0x00000d70 0x08000d70 - 0x08000dff --- fmap..rela.eh_frame
+ 2 fd: 3 +0x00000920 0x08000920 - 0x08000aff --- fmap..symtab
+ 1 fd: 3 +0x00000000 0x08000000 - 0x0800003f r-- fmap.ehdr
+--
+            0x08000079      e800000000     call  strcmp
+            0x080000a2      e800000000     call  strcmp
 EOF
 RUN
 
 NAME=ELF: reloc flags
 FILE=bins/elf/analysis/filetime.c-clang-x64-O0.o
```

Comment:
Could you please add also the [rawmem.c-gcc-x64-O3.o](https://github.com/rizinorg/rz-ghidra/files/6120419/rawmem.c-gcc-x64-O3.o.zip) from https://github.com/rizinorg/rz-ghidra/issues/203 to tests?

Reply:
Will add it. Are there any specific differences to the filetime one that I should test for especially?

Reply:
Oh there is a nice jumptable that is built on relocs, I will add a test for that.